lib/record/s3.js

```diff
@@ -0,0 +1,84 @@
+const Account = require('../models/account');
+const Websocket = require('ws');
+const PCMToMP3Encoder = require('./encoder');
+const S3MultipartUploadStream = require('./s3-multipart-upload-stream');
+const wav = require('wav');
+
+async function upload(logger, socket) {
+
+  socket.on('message', async function(data, isBinary) {
+    try {
+      if (!isBinary) {
+        const obj = JSON.parse(data.toString());
+        logger.info({obj}, 'received JSON message from jambonz');
+        const {sampleRate, accountSid, callSid} = obj;
```

Comment:
In addition to these properties, I would also like to create metadata properties in the S3 uploaded file for:
- direction
- from
- to 
- callId
- applicationSid
- originatingSipIp
- originatingSipTrunkName